src/touch.js

```diff
@@ -31,20 +32,18 @@ function _touch(opts, files) {
     'r': 'reference',
   });
 
-  if (!files) {
-    common.error('no paths given');
-  }
-
-  if (Array.isArray(files)) {
-    files.forEach(function(f) {
-      touchFile(opts, f);
-    });
-  } else if (typeof files === 'string') {
-    touchFile(opts, files);
-  } else {
+  if (!files)
+    common.error('no files given');
+  else if (Array.isArray(files))
```

Comment:
This is to do a no-op. For what it's worth, I think #352 will make this line obsolete, so we might want to wait until that's merged